composer.json

```diff
@@ -1,32 +1,35 @@
 {
-  "name": "cwp/cwp-installer",
-  "description": "CWP Project Template",
-  "license": "BSD-3-Clause",
-  "require": {
-    "cwp/cwp-recipe-basic": "1.8.x-dev",
-    "cwp/cwp-recipe-blog": "1.8.x-dev",
-    "cwp/starter-theme": "1.1.x-dev"
-  },
-  "require-dev": {
-    "cwp/cwp-recipe-basic-dev": "^1.1@stable"
-  },
-  "suggest": {
-    "silverstripe/textextraction": "Required for document text searching within CWP"
-  },
-  "config": {
-    "process-timeout": 900
-  },
-  "extra": {
-    "branch-alias": {
-      "dev-master": "1.8.x-dev"
-    }
-  },
-  "repositories": [
-    {
-      "type": "composer",
-      "url": "https://packages.cwp.govt.nz/"
-    }
-  ],
-  "prefer-stable": true,
-  "minimum-stability": "dev"
+    "name": "cwp/cwp-installer",
+    "description": "CWP Project Template",
+    "license": "BSD-3-Clause",
+    "require": {
+        "cwp/cwp-recipe-core": "2.x-dev",
+        "cwp/starter-theme": "2.x-dev"
+    },
+    "extra": {
+        "project-files": [
+            "mysite/*"
+        ],
+        "branch-alias": {
+            "dev-master": "2.x-dev"
+        },
+        "project-files-installed": [
+            "mysite/.htaccess",
+            "mysite/_config.php",
+            "mysite/_config/environmentcheck.yml",
+            "mysite/_config/mysite.yml"
+        ],
```

Comment:
This part is generated by installing a project, but it shouldn't be committed in here